packages/lesmis/src/components/MapOverlaysPanel/MapOverlaysPanel.js

```diff
@@ -69,35 +70,37 @@ export const MapOverlaysPanel = ({
   ]);
 
   return (
```

Comment:
the only change here is wrapping everything with `ErrorBoundary`

Reply:
The error boundary will show an error message if there are any unhandled errors further down the component tree rather than the whole app crashing.